coriolis/osmorphing/osmount/windows.py

```diff
@@ -105,6 +110,30 @@ def _service_disks_with_status(
                             raise
                     break
 
+    def _get_disk_ids_from_drive_letters(self, drive_letters):
+        disk_ids = []
+        volume_entry_re = r"\s+Volume ([0-9]+)\s+([A-Z]+)\s+"
```

Comment:
Nit: in such cases where we're after a very specific match it's always best to make regexes as specific as possible.
Considering the 'Label' in the `LIST VOLUME` can be whatever it wants, please use a leading '^' for the regex so we always catch the beginning of the line.